tests/Setup.php

```diff
@@ -19,8 +20,8 @@ public static function doSetup()
     {
         Configuration::reset();
 
-        Configuration::environment('development');
-        Configuration::publicKey('integration_public_key');
-        Configuration::privateKey('integration_private_key');
+        Configuration::environment('production');
+        Configuration::publicKey('ASC7AsydNKAKBB5BNEVFVZ0P');
+        Configuration::privateKey('4vapebxbpq26gmk7a2zdj837u0qn24ufynqnnjfq');
```

Comment:
@Sheikh-Aman Red flag for showing private key and public key in a public repo....

Reply:
Thanks @Yongyiw , fixed this.